lib/mrsk/cli/app.rb

```diff
@@ -76,7 +76,10 @@ def details
   desc "exec [CMD]", "Execute a custom command on servers (use --help to show options)"
   option :interactive, aliases: "-i", type: :boolean, default: false, desc: "Execute command over ssh for an interactive shell (use for console/bash)"
   option :reuse, type: :boolean, default: false, desc: "Reuse currently running container instead of starting a new one"
+  option :labels, aliases: "-l", type: :array, default: [], desc: "Labels to apply to the container (not compatible with --reuse)", banner: "label1=value1 label2=value2 label3"
```

Comment:
Thor option arrays are greedy for parameters that follow the option flag, so you can't have the command listed straight after them, because it will treat it as a label.

So instead of:
```
$ mrsk app exec -i -l foo=bar bash
ERROR: "mrsk app exec" was called with no arguments
Usage: "mrsk app exec [CMD]"
```

You'd need one of:
```
mrsk app exec bash -i -l foo=bar
mrsk app exec -i bash -l foo=bar
mrsk app exec -l foo=bar -i bash
```